src/main/java/teammates/common/util/Const.java

```diff
@@ -794,6 +794,8 @@ public static final String getjQueryUiFilePath(boolean isDevServer) {
         public static final String EMAIL_COURSE = "course";
         public static final String EMAIL_FEEDBACK = "feedback";
         public static final String EMAIL_TYPE = "type";
+        public static final String EMAIL_SEND_CHECK = "sessionsummarysendemail";
```

Comment:
`EMAIL_SEND_CHECK` is quite vague. If it is specifically about session summary, it should be reflected in the variable name.